co_sim_io/impl/info.hpp

```diff
@@ -145,6 +145,11 @@ class Info
         mOptions[I_Key] = std::make_shared<Internals::InfoData<TDataType>>(I_Value);
     }
 
+    void Set(const std::string& I_Key, const char * I_Value)
```

Comment:
I am still not sure this is needed
I even added a test where I pass a char and this works